src/test/java/net/snowflake/client/authentication/OktaAuthLatestIT.java

```diff
@@ -0,0 +1,74 @@
+package net.snowflake.client.authentication;
+
+import static net.snowflake.client.authentication.AuthConnectionParameters.SSO_USER;
+import static net.snowflake.client.authentication.AuthConnectionParameters.getOktaConnectionParameters;
+
+import java.io.IOException;
+import java.util.Properties;
+import net.snowflake.client.category.TestTags;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag(TestTags.AUTHENTICATION)
+class OktaAuthLatestIT {
+
+  AuthTest authTest;
```

Comment:
AuthTest does not contain any tests - please rename the class to be helper

Reply:
done